tailwind.config.mjs

```diff
@@ -16,6 +16,7 @@ export default {
       animation: {
         adderPopUp: 'adderPopUp 0.08s ease-in forwards',
         adderPopDown: 'adderPopDown 0.08s ease-in forwards',
+        'fade-in-slow': 'fade-in 1s ease-in',
```

Comment:
I noticed when adding this animation that I hadn't been following TW's convention of kebab-case for...pretty much everything. I will update `adderPopUp` to `adder-pop-up`, etc., in this config as a bit of follow-up work.